styles/components/_modalWindow.scss

```diff
@@ -0,0 +1,159 @@
+// Positioning /
+// Display & Box Model /
+// Color /
+// Text /
+// Other /
+
+.modal {
+    // Positioning /
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    z-index: 1;
+    // Display & Box Model /
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    // Color /
+    background-color: rgba(0,0,0,.8);
+    &__container {
+        // Display & Box Model /
+        height: 889px;
+        width: 1200px;
+        border-radius: 16px;
+        padding: 3rem 4rem;
+        display: flex;
+        flex-direction: column;
+        // Color /
+        background: #FFFFFF;
+        box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+        // Text /
+        text-align: left;
+    }
+    &__heading {
+        margin: 0 auto;
+        text-align: center;
+        text-transform: uppercase;
+        line-height: 51px;
+    }
+    &__text {
+        margin: 1rem 0;
+        font-weight: 500;
+        font-size: 28px;
+        line-height: 34px;
+    }
+}
+
+.details {
+    margin-left: 285px;
+    height: 355px;
+    padding-top: .75rem;
+    &__heading {
+        margin-bottom: .2rem;
+        margin-left: 50px;
+        font-weight: 600;
+        font-size: 26px;
+    }
+    &__table {
+        height: 40%;
+        font-size: 24px;
+        font-weight: 600;
+        & tbody, & thead tr {
+            display: block;
+        }
+        & thead {
+            padding-left: 180px;
+        }
+        & tbody {
+            height: 250px;
+            border-spacing: 1rem 2rem ;
+            overflow-y: scroll;
+            overflow-x: hidden;
+            vertical-align: middle;
+        }
+        & th:first-of-type, & td:first-of-type {
+            width: 125px;
+            padding: 0 30px;
+        }
+        & th:first-of-type {
+            width: 157px;
+        }
+        & tbody td, & thead th {
+            width: 206px;
+        }
+        &__head {
+            font-weight: 600;
+            color: rgba(0, 0, 0, 0.6);
+        }
+        &__image {
+            position: relative;
+            & img {
+                z-index: 1;
+                width: 65px;
+                height: auto;
+                border-radius: $border-radius;
+                transform: matrix(-1, 0, 0, 1, 0, 0);
+                filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25)) drop-shadow(4px 4px 20px rgba(255, 0, 0, 0.9));
+            }
+        }
+        &__answer {
+            color: $text-color-primary;
+            &--correct {
+                color: rgba(87, 181, 13, 0.6);
+            }
+            &--incorrect {
+                color: rgba(255, 0, 0, 0.6);
+            }
+        }
+    }
+    &__image{
+        height: 3rem;
+        width: auto;
+        overflow: hidden;
+        &__container {
+            width: 50px;
+        }
+    }
+}
+
+.player-form {
+    display: flex;
+    flex-direction: row;
```

Comment:
It's row by default right?
```suggestion

```

Reply:
I usually wrote it so I knew without having to wonder what it was. I am not a big fan of 'implicit'.